docs/create/advanced/troubleshooting.mdx

```diff
@@ -20,7 +20,7 @@ Here you'll find instructions for troubleshooting common issues and problems tha
 
 ### Process exited with code 1 (ChildProcess)
 
-There are times during development where you may encounter vague error responses from the developer CLI. This can leave you wondering how to proceed, because the error response itself doesn't explain what part of the command failed. These errors are usually triggered by Node's [Child Process](https://nodejs.org/api/child_process.html) module panicking and failing to complete a [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise).
+There are times during development where you may recieve unclear error responses from the developer CLI. This can be because the error response itself doesn't explain what part of the command failed. These errors are usually triggered by Node's [Child Process](https://nodejs.org/api/child_process.html) module panicking and failing to complete a [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise).
```

Comment:
```suggestion
There are times during development where you may receive unclear error responses from the developer CLI. This can be because the error response itself doesn't explain what part of the command failed. These errors are usually triggered by Node's [Child Process](https://nodejs.org/api/child_process.html) module panicking and failing to complete a [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise).
```